code/game/shuttle_engines.dm

```diff
@@ -154,7 +154,7 @@
 	desc = "An extremely large bluespace engine used to propel extremely large ships."
 	bound_width = 96
 	bound_height = 96
-	appearance_flags = 0
+	appearance_flags = LONG_GLIDE
```

Comment:
shouldn't thing that do `appearance_flags = NONE` also be changed then? as NONE = 0